lms/static/scripts/frontend_apps/components/dashboard/SyncGradesButton.tsx

```diff
@@ -58,6 +55,20 @@ export default function SyncGradesButton({
     [lastSync],
   );
 
+  const syncedStudentsCount = useMemo(
+    () =>
+      lastSync.data?.grades.filter(s => s.status !== 'in_progress').length ?? 0,
+    [lastSync.data?.grades],
+  );
```

Comment:
I assume that `lastSync.data.grades` is always a full list which is never paginated?

Reply:
Yes.